### ISSUE TITLE
[FlexibleColumnLayout][Accessibility] Missing title and aria-label for collapse button

### ISSUE THREAD

**Issue by Neeeko** [SAP/ui5-webcomponents#2226]
I'm part of the **SuccessFactors** team.

**Describe the bug**
The collapse/dismiss button of the ui5-flexible-column-layout is missing a title or at the very least an aria-label for screen readers which is not accessible.

**Expected behavior**
Maybe add an attribute to ui5-flexible-column-layout to allow developers to add an aria-label to this button. (something similar to this: https://github.com/SAP/ui5-webcomponents/issues/2187)

**Screenshots**
![Screen Shot 2020-09-17 at 16 48 38](https://user-images.githubusercontent.com/7316793/93487770-e330cb80-f905-11ea-894b-4f2dbd6775e0.png)

**Context**
- UI5 Web Components version: 0.23.2
- OS/Platform: macOS Catalina

**Affected components** *ui5-flexible-column-layout*

**Comment by ilhan007** [comment]
I think this older issue by sf team requests an option to set aria-label from outside
https://github.com/SAP/ui5-webcomponents/issues/2183

If we provide a title or internal aria-label for those arrow buttons, would you still need an option to set an aria-label from outside?

**Comment by Neeeko** [comment]
> I think this older issue by sf team requests an option to set aria-label from outside
> #2183
> 
> If we provide a title or internal aria-label for those arrow buttons, would you still need an option to set an aria-label from outside?

I didn't notice the existing issue which is indeed similar.
Sorry for the inconvenience!
I would be fine with an internal aria-label. Please see with Hubery on the other issue if it fits with his requirements (we don't work on the same project!) 🙂
